core/src/test/java/org/apache/iceberg/util/TestParallelIterable.java

```diff
@@ -0,0 +1,72 @@
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
can you run `./gradlew spotlessApply` to get the license header fixed?

Reply:
Sorry I forget run `./gradlew :iceberg-core:spotlessApply`.